common/djangoapps/third_party_auth/management/commands/saml.py

```diff
@@ -16,9 +15,6 @@ def add_arguments(self, parser):
         parser.add_argument('--pull', action='store_true', help="Pull updated metadata from external IDPs")
 
     def handle(self, *args, **options):
-        if not SAMLConfiguration.is_enabled():
```

Comment:
Yeah, it was a separate hotfix that went onto the rc this cycle: https://github.com/edx/edx-platform/pull/13634